rcp-disable-site-wide-gateway-scripts.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Plugin Name: Restrict Content Pro - Disable Site-Wide Gateway Scripts
+ * Description: Ensures gateway scripts only load on Restrict Content Pro pages. This primarily prevents stripe.js from loading across the whole site.
+ * Version: 1.0
+ * Author: Restrict Content Pro team
+ * License: GPL2
+ */
+
+/**
+ * Disables gateway scripts like Stripe.js if the current page is not an RCP page.
+ *
+ * @return void
+ */
+function ag_rcp_maybe_disable_gateway_scripts() {
+
+	// Allow on registration page.
+	if ( rcp_is_registration_page() ) {
```

Comment:
We should check that this function exists before using it. If someone deactivates RCP briefly the site would fatal. I just had the realization that since all these snippets are now plugins, we should probably check them all for this.

Reply:
Good thinking. 👍